finalPrep/modZeroHero.js

```diff
@@ -1,41 +1,59 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Cal the Calmer"
+var specialAbility = "Chi Manipulation"
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
-
+var greeting = "I am" + heroName + "and I'm here to bring you down a notch!";
+var catchphrase = "$(specialAbility) for any situation!";
 // Declare two variables - power AND energy - set to integers
-
+var power = 1000;
+var energy = 10;
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
-
+var fullPower = (power * 500);
+var energy = (energy + 150);
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true;
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ["Karen the Complicator", "Undies in a Bundle", "Steve the Stressman"];
+var sidekicks = ["Chilled out Charlie", "Relaxing Rex", "Slow Down Sally"];
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemy[2]);
 // Write some code to add a new archEnemy to the archEnemies array
-
+archEnemies.push("Tense Travis");
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemies);
 // Remove the first sidekick from the sidekicks array
-
+sidekicks.shift();
 // Print the sidekicks array to console to ensure you added a new sidekick
-
+console.log(sidekicks);
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
-
+var dangerLevel = 20;
+var saveTheDay = "I am here to save the day!";
+var badExcuse = "I just remembered I have cookies in the oven..."
+
+function assessSituation (dangerLevel, saveTheDay, badExcuse){
+  if (dangerLevel > 50){
+    console.log(badExcuse);
+  } else if (dangerLevel >=10 && dangerLevel <=50);
```

Comment:
Since line 49 already checks for dangerLevel greater than 50, line 51 could simply check for dangerLevel > 9, but this works too!